metrics/repl.lag/lag.go

```diff
@@ -16,32 +17,65 @@ import (
 const (
 	DOMAIN = "repl.lag"
 
-	OPT_SOURCE_ID            = "source-id"
-	OPT_SOURCE_ROLE          = "source-role"
-	OPT_TABLE                = "table"
-	OPT_WRITER               = "writer"
-	OPT_REPL_CHECK           = "repl-check"
-	OPT_REPORT_NO_HEARTBEAT  = "report-no-heartbeat"
-	OPT_REPORT_NOT_A_REPLICA = "report-not-a-replica"
-	OPT_NETWORK_LATENCY      = "network-latency"
+	OPT_HEARTBEAT_SOURCE_ID                = "source-id"
+	OPT_HEARTBEAT_SOURCE_ROLE              = "source-role"
+	OPT_HEARTBEAT_TABLE                    = "table"
+	OPT_WRITER                             = "writer"
+	OPT_REPL_CHECK                         = "repl-check"
+	OPT_REPORT_NO_HEARTBEAT                = "report-no-heartbeat"
+	OPT_REPORT_NOT_A_REPLICA               = "report-not-a-replica"
+	OPT_RENAME_DEFAULT_REPLICATION_CHANNEL = "rename-default-replication-channel"
```

Comment:
`default-channel-name` (default: ""), then Blip and MySQL line up: the default channel name is the same when not specified: an empty string.

Reply:
Also, intention was to let user rename it whatever, not to force a new default of "default". E.g. a user might want to `default-channel-name: my-primary-db` or something.